rockcraft.yaml

```diff
@@ -8,16 +8,17 @@ base: ubuntu:22.04
 platforms:
   amd64:
 
-entrypoint: ["/bin/bash", "/opt/pebble-start.sh"]
+entrypoint: ["/bin/bash", "/opt/pebble/pebble-start.sh"]
 
 env:
   - SPARK_HOME: /opt/spark
+  - SPARK_CONFS: /etc/spark8t/conf
```

Comment:
**thought** This specification is diverging a bit from the snap, namely in SNAP we picked `/etc/spark-client/conf`, we should probably keep this consistent. Actually maybe this choice is better than the one we picked in the SNAP, and we should revert that bit. Let's discuss this later

Reply:
https://warthogs.atlassian.net/browse/DPE-2062